### PR TITLE
Todo List 

### PR DESCRIPTION
**Pull request by samiullahbahadur**
# Todo list structure

- I create webpack 
- used ES6
- Create item and remove item

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  constructor(desp, arr, done = false) {
+    this.desp = desp;
+    this.index = arr.length;
+    this.done = done;
+  }
+}
+
+class TaskLisk {
+  constructor() {
+    this.tasks = [];
```
Comment (FrederickMih) [inline src/index.js:17]: - [ ] Hi @samiullahbahadur, I can see that you are doing so great in this project :+1:  but from the requirements of this particular project of todo list app, tasks arrays should contain some simple to do tasks (array of objects) containing three keys: description, completed, and index. Please let's fix this.

Comment (FrederickMih) [inline src/index.js:17]: - [ ] Again, from the requirements of this project, the task array should be rendered dynamically to the DOM on page load.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
#### Hi @samiullahbahadur , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight

:arrow_forward: Git/Github workflow has been followed :heavy_check_mark: 
:arrow_forward: Linter has been set up correctly :heavy_check_mark: 
:arrow_forward: Good commit history :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by samiullahbahadur** [comment]
okay, I fix it

**Comment by omar-labana** [comment]
Hey @samiullahbahadur, what was the error mentioned by the previous reviewer? I can't seem to find his comment.

**Comment by samiullahbahadur** [comment]
Nothings ,  just about readme i fix it
But unfortunite remove it 
You check the project.
If found any other issue then tell me i will do fix

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @samiullahbahadur,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**